App/Containers/PhotoViewerScreen.js

```diff
@@ -85,8 +97,10 @@ class PhotoViewerScreen extends React.PureComponent {
 const mapStateToProps = (state, ownProps) => {
   const hashes = state.ipfs.threads[ownProps.navigation.state.params.thread].hashes
   const imageData = hashes.map(hash => {
-    return { hash, source: { uri: 'https://localhost:9080/ipfs/' + hash + '/photo' } }
+    // todo, try source here again
+    return { hash, source: require('../Images/TextileHeader.png')}
```

Comment:
You can just put in a fake url for source so it doesn't actually try to load this image resource when we're not even using it.

Reply:
Just put `file://foo.jpg` or something.

Reply:
done

Reply:
any invalid placeholder messes it up. this was simply to avoid this warning,

![screen shot 2018-05-08 at 5 55 38 pm](https://user-images.githubusercontent.com/370259/39790145-17b149b8-52e9-11e8-964e-fa97d15fee2f.png)

maybe if we don't care about that warn i can just ignore it

Reply:
hmm, i think as long as it's a valid source object (`{uri: string}`) it should be fine. That's also something you can see in the base64 commits

Reply:
so it does... fixed